src/streaming.jl

```diff
@@ -215,7 +215,7 @@ function parsefile(filename::AbstractString,callbacks::XPCallbacks; bufferlines=
                 write(io, readline(file))
                 i += 1
             end
-            txt = Compat.bytestring(io)
+            txt = Compat.String(io)
```

Comment:
It says `bytestring` has been replaced by additional `String` methods. For v0.3 compatibility, one should replace `bytestring` by  `@compat String`. That doesn't work for v0.4 though.